app/src/main/res/xml/prefactions_share_into.xml

```diff
@@ -40,7 +40,6 @@
         android:title="@string/calendar" />
 
     <PreferenceCategory
-        android:key="@string/pref_key__favourite_files"
```

Comment:
intentional?

Reply:
Yes. Android studio was complaining about there being a duplicate "perf_key__favourite_files". I don't think removing this second one hurts anything.